azure-pipelines.yml

```diff
@@ -128,6 +128,13 @@ jobs:
     displayName: 'Check installed packages'
   - powershell: |
       cd testsuite
+      $env:OPENBLAS_NUM_THREADS=1
+      $env:GOTO_NUM_THREADS=1
+      $env:OMP_NUM_THREADS=1
+      $env:MKL_NUM_THREADS=1
+      # limit to 2 workers to avoid overloading the CI
+      $env:PYTEST_XDIST_AUTO_NUM_WORKERS=1
```

Comment:
The comment is confusing: if you set workers to 1 but say you limit to 2 workers. Change comment, otherwise same as above: perhaps just add to commandline args?

Reply:
I forgot to correct the comment line when I realized Azure has only 2 cores so I changed # workers to 1.

Reply:
As above, is this actually affecting timeouts or is this some separate optimization?